bedrock/mozorg/templates/mozorg/moss/mission-partners.html

```diff
@@ -0,0 +1,319 @@
+  <h2 class="moss-section-heading">{{ _('How we’ll make our decision') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    All criteria are indicative rather than determinative — that is to say, they
+    will make us more or less likely to make an award, but none will guarantee a
+    particular outcome.
+  {% endtrans %}
+  </p>
+
+  <p class="moss-copy">
+    {{ _('Selection criteria:') }}
+  </p>
+
+  <ul class="moss-bullet-list">
+    <li>
+      <p class="moss-copy">
+      {% trans %}
+        How significantly does the project advance the Mozilla mission?
+      {% endtrans %}
+      </p>
+    </li>
+    <li>
+      <p class="moss-copy">
+      {% trans %}
+        What sort of reputation does the project have in general, if any?
+      {% endtrans %}
+      </p>
+    </li>
+    <li>
+      <p class="moss-copy">
+      {% trans %}
+        This includes reputation in technical, inclusion and other areas
+      {% endtrans %}
+      </p>
+    </li>
+    <li>
+      <p class="moss-copy">
+      {% trans %}
+        Is the project known for something besides the code we are relying on?
+      {% endtrans %}
+      </p>
+    </li>
+    <li>
+      <p class="moss-copy">
+        {{ _('Will this award make a significant impact on the project?') }}
+      </p>
+    </li>
+    <li>
+      <p class="moss-copy">
+      {% trans %}
+        Is the level of funding appropriate for the task to be accomplished?
+      {% endtrans %}
+      </p>
+    </li>
+    <li>
+      <p class="moss-copy">
+      {% trans %}
+        Does the person (or group) who will receive the money have a track
+        record of delivering?
+      {% endtrans %}
+      </p>
+    </li>
+  </ul>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('How to apply') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    To get started with your application, simply follow the link below.
+  {% endtrans %}
+
+  {% trans url1='https://wiki.mozilla.org/MOSS#Application_Deadlines' %}
+    For deadline information, please check
+    <a href="{{ url1 }}">the MOSS wiki</a>.
+  {% endtrans %}
+
+  {% trans url1='https://wiki.mozilla.org/MOSS/Mentors' %}
+    If you need help completing your application, see the
+    <a href="{{ url1 }}">list of mentors</a>.
+  {% endtrans %}
+  </p>
+
+  <div class="moss-ctas">
+    <p class="moss-copy">
+      I’m ready!
+    </p>
+
+    <a href="https://docs.google.com/forms/d/1rwYQTT-9-eldS-kElY646bMwMzJpxfL8lDskX86xgCQ/viewform" rel="external" class="button">{{ _('Apply Now') }}</a>
+  </div>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('FAQs') }}</h2>
+
+  <dl class="moss-dl">
+    <dt class="moss-copy">
+      {{ _('What if my project doesn’t have a legal organizational home?') }}
+
+    {% trans %}
+      I see this is disfavored, and exceptional circumstances are required.
+    {% endtrans %}
+
+      {{ _('What might count as “exceptional”?') }}
```

Comment:
Yeah, I think maybe you did go a little overboard with the "no long lines" stuff 😉 I'm finding the switching between `{% trans %}` and `{{ _(''} }}` within the same paragraph less readable than a consistently formatted stack of long lines. I think it might be better to pick one method for enclosing strings and stick with it, at least within a given block of text.